chapters/dashboard/hub.md

```diff
@@ -0,0 +1,14 @@
+# Jina Hub
+
+**[Jina Hub](https://dashboard.jina.ai/#/hub)** is a centralized registry, where you can share and discover custom and community driven Jina Pods and Apps tailored to specific use cases of neural search.
+
+# Features
+
+-  ##  Upload and Share Images
```

Comment:
use either headings or bullet points, it looks kind of wierd like this

Reply:
changed it to headings